src/main/java/org/snomed/snowstorm/core/data/domain/ConcreteValue.java

```diff
@@ -0,0 +1,66 @@
+package org.snomed.snowstorm.core.data.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonView;
+import org.snomed.snowstorm.rest.View;
+import org.springframework.data.annotation.Transient;
+
+/**
+ * Represent the Value of a Concrete Relationship
+ * in API response.
+ */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+@JsonView(value = View.Component.class)
+public final class ConcreteValue {
```

Comment:
No need to mark classes as final. In general Snowstorm is not used as a library but we know that some implementers do extend it for wrapping into an OSGi bundle and other purposes, we don't want to block them extending our classes for whatever reason.

Reply:
Fixed.